src/index.test.js

```diff
@@ -61,4 +57,64 @@ describe('CGUs', () => {
       expect(resultingTerms).to.be.equal(SERVICE_B_TOS_VERSION);
     });
   });
+
+  describe('#refilterAndRecord', () => {
+    context('When a service’s filter declaration change', () => {
+      let originalSnapshotId;
+      let firstVersionId;
+      let refilterVersionId;
+      let refilterVersionMessageBody;
+      let serviceBVersionId;
+
+      before(async () => {
+        nock('https://www.servicea.example').get('/tos').reply(200, SERVICE_A_TOS_SNAPSHOT);
+        nock('https://www.serviceb.example').get('/tos').reply(200, SERVICE_B_TOS_SNAPSHOT);
+        const app = new CGUs();
+        await app.init();
+        await app.trackChanges();
+
+        const [originalSnapshotCommit] = await gitSnapshot().log({ file: SERVICE_A_EXPECTED_SNAPSHOT_FILE_PATH });
+        originalSnapshotId = originalSnapshotCommit.hash;
+
+        const [firstVersionCommit] = await gitVersion().log({ file: SERVICE_A_EXPECTED_VERSION_FILE_PATH });
```

Comment:
I thought ESlint was supposed to catch this 🤔 
```suggestion
        const [ firstVersionCommit ] = await gitVersion().log({ file: SERVICE_A_EXPECTED_VERSION_FILE_PATH });
```